docs/unleash-context.md

```diff
@@ -0,0 +1,22 @@
+# Unleash Context
+
+In order to standardize a few activation strategies we also needed to 
```

Comment:
I'm not sure if you aim for British or American English, but this is the American spelling. On purpose?